service/src/test/java/uk/gov/hmcts/reform/fpl/service/NoticeOfProceedingsServiceTest.java

```diff
@@ -27,14 +30,26 @@
 @ExtendWith(SpringExtension.class)
 @SpringBootTest(classes = {NoticeOfProceedingsService.class})
 @ContextConfiguration(classes = {
-    JacksonAutoConfiguration.class, LookupTestConfig.class, HearingBookingService.class,
-    HearingVenueLookUpService.class, CaseDataExtractionService.class, FixedTimeConfiguration.class
+    JacksonAutoConfiguration.class,
+    LookupTestConfig.class,
+    HearingBookingService.class,
+    HearingVenueLookUpService.class,
+    CaseDataExtractionService.class,
+    FixedTimeConfiguration.class,
+    NoticeOfHearingGenerationService.class,
+    HearingVenueLookUpService.class
 })
 class NoticeOfProceedingsServiceTest {
 
     @Autowired
     private NoticeOfProceedingsService noticeOfProceedingService;
 
+    @MockBean
+    private DocmosisDocumentGeneratorService docmosisDocumentGeneratorService;
+
+    @MockBean
+    private UploadDocumentService uploadDocumentService;
+
```

Comment:
I will stop commenting after this, but I would suggest a nicer approach instead of adding dependencies to the test which we don't use, seems we are doing something wrong here.